htslib/faidx.h

```diff
@@ -357,6 +357,15 @@ int fai_adjust_region(const faidx_t *fai, int tid,
 HTSLIB_EXPORT
 void fai_set_cache_size(faidx_t *fai, int cache_size);
 
+/// Adds a thread pool to the underlying BGZF layer.
+/** @param fai         FAI file handler
+ *  @param pool        The thread pool (see hts_create_threads)
+ *  @param qsize       The size of the job queue.  If 0 this is twice the
+ *                     number of threads in the pool.
+ */
+HTSLIB_EXPORT
+int fai_thread_pool(faidx_t *fai, struct hts_tpool *pool, int qsize);
```

Comment:
It currently gets it as a byproduct of _hts.h_, but it would probably be worth adding a `struct hts_tpool;` declaration above (beside the other struct declarations).

Reply:
Easy enough to do. Thanks